### ISSUE TITLE
 No PHP classes detected for files

### ISSUE THREAD

**Issue by norgeindian** [yireo/Yireo_ExtensionChecker#28]
We have a module, which does not include any PHP files.
When running the `scan` command on all of our modules, the process stops at this point with the following output:

```
+---------+-------+------------+
| Message | Group | Suggestion |
+---------+-------+------------+
```
In verbose mode, we get the following error:
```
+-----------------------------------+-----------+------------+
| Message                           | Group     | Suggestion |
+-----------------------------------+-----------+------------+
| No PHP classes detected for files | Exception |            |
+-----------------------------------+-----------+------------+
```
In my eyes, this should not return an error.

**Comment by jissereitsma** [comment]
This felt like a good idea at first. But your case shows it is not. Also, there are Integration Tests already in place to check for this anyway. Fixed in the next release: https://github.com/yireo/Yireo_ExtensionChecker/commit/04966d357c9e892b0dbb0d5744399306d1352cb9

**Comment by jissereitsma** [comment]
This should now be fixed in 2.1.2.
